config/locales/es.yml

```diff
@@ -2715,6 +2715,14 @@ es:
           primary_disabled_question: "¿Está usted discapacitado según la definición de Nueva Jersey?"
           spouse_disabled_question: "¿Su cónyuge está discapacitado según la definición de Nueva Jersey?"
           title: Exención para discapacitados
+      nj_estimated_tax_payments:
+        edit:
+          label: Total
+          subtitle: 'Please include the sum of the following:'
+          subtitle_list_1: Estimated income tax payments you made throughout the year for your New Jersey state taxes
+          subtitle_list_2: Overpayments on your %{prior_year} return that you chose to apply to your %{filing_year} New Jersey taxes
+          subtitle_list_2_sub: 'Note: you would have selected this option when filing your %{prior_year} return'
+          title: Please tell us if you already made any payments toward your %{filing_year} New Jersey taxes
```

Comment:
Do we have Spanish content to put into this file?

Reply:
We agreed we are not putting in any placeholder spanish content for now. No need to take the effort of running it through Google Translate since it will all be replaced with actual content once Barbi does her full content review